sdl_viewer/src/camera.rs

```diff
@@ -207,13 +210,13 @@ impl Camera {
     }
 
     pub fn pan(&mut self, x: f32, y: f32, z: f32) {
```

Comment:
There are duplicated functions for rotation and panning:
-  mouse_drag_pan() and pan()
-  mouse_drag_rotate() and rotate(). 

The mouse_* functions use constants (100) inside of the function while the non-mouse functions use constants outside of the function. Could we unify this to two function such as rotate(up, around) and pan(x,y,z) and set all  constants from the call site? Specifically, I find it inconsistent that Joystick computes the constants at the call site while mouse does not.

Reply:
Created a ticket to follow-up on it as a clean-up task